x509/build/build.rs

```diff
@@ -106,6 +106,6 @@ fn gen_rt_alias_cert(out_dir: &str) {
                 digest: &[0xCD; 48],
             },
         }]);
-    let template = bldr.tbs_template("Caliptra Rt Alias", "Caliptra FMC Alias");
+    let template = bldr.tbs_template("Caliptra 1.0 Rt Alias", "Caliptra 1.0 FMC Alias");
```

Comment:
You'll also need to update this to match: https://github.com/chipsalliance/caliptra-sw/blob/main/runtime/src/dpe_platform.rs#L80

Reply:
Fixed.